docs/src/pages/customization/breakpoints/WithWidth.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import withWidth from '@material-ui/core/withWidth';
 import Typography from '@material-ui/core/Typography';
-import { Breakpoint } from '@material-ui/core/styles/createBreakpoints';
 
+type Breakpoint = 'xs' | 'sm' | 'md' | 'lg' | 'xl';
```

Comment:
Didn't we have a dedicated type for this that is public? Hard-coding means [custom breakpoints via module augmentation](https://next--material-ui.netlify.app/customization/breakpoints/#custom-breakpoints) doesn't work.

Reply:
We raise against

```tsx
import { Breakpoint } from '@material-ui/core/styles/createBreakpoints';
```

in https://github.com/mui-org/material-ui/blob/1691d498b3c7fc8c7f4337948e7ad7c2947f66a2/.eslintrc.js#L44

I don't have any preference

Reply:
I understand. But we should have a public type for it.

Reply:
Should I disable the rule for these cases?

Reply:
I will update the eslint rule until we resolve what we want to do with it.